.github/actions/test-notify-slack/action.yml

```diff
@@ -0,0 +1,74 @@
+---
+
+name: 'Test notify slack action'
+description: 'Runs validation against the notify-slack action'
+
+inputs:
+  application:
+    description: 'The friendly name for the application'
+    type: string
+    required: true
+  data:
+    description: 'The path to the file with data to pass to the render'
+    type: string
+    required: false
+    default: ''
+  environment:
+    description: |
+      The environment variables to pass to the render. Each environment variable should be in format VAR=VAL with one on each line.
+      Example:
+        environment: |
+          FOO=bar
+          BAR=baz
+    type: string
+    required: false
+    default: ''
+  format:
+    description: 'The format the data file will be in'
+    type: string
+    required: false
+    default: ''
+  version:
+    description: 'The version that is being deployed'
+    type: string
+    default: ${{ github.ref }}
+  type:
+    description: 'The type of action that is being performed such as build or deployment'
+    type: string
+    required: true
+  template:
+    description: 'The path to the template file to render'
+    type: string
+    required: true
+  status:
+    description: 'The status of the deployment.  Options are started, success, and failure'
+    type: string
+    required: true
+  slack-webhook-url:
+    description: 'The slack webhook url to post the message to'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: 'Checkout'
+      uses: actions/checkout@v3
+
+    - name: 'Render payload'
+      uses: shopsmart/render-j2-action@v1
+      with:
+        template: ${{ github.action_path }}/message.json.j2
+        data: ${{ inputs.data }}
+        format: json
+
+    - name: 'Post message to slack'
+      id: slack
+      uses: ./actions/notify-slack
+      with:
+        application: ${{ inputs.application }}
+        environment: ${{ inputs.environment }}
+        status: ${{ inputs.status }}
+        type: ${{ inputs.deployment }}
+        version: ${{ inputs.version }}
+        template: ${{ inputs.template }}
+        slack-webhook-url: ${{ inputs.slack-webhook-url }}
```

Comment:
This action is currently wrapping the notify-slack action and is not really necessary.  This action is meant to call the notify-slack action and then make any assertions.  In this case, we can simply call the notify-slack with hardcoded values.